frontend/src/console/reducers.ts

```diff
@@ -10,6 +10,18 @@ export function consoleReducer(
   let activeInstanceId: number;
 
   switch (action.type) {
+    case Actions.Types.USER_REFRESH_DATA:
+      return update(state, {
```

Comment:
You should use the clean state defined on `models.ts` instead of manually cleaning up all variables.
See `initialConsoleState` on `console/models.js`:
https://github.com/open-craft/opencraft/blob/b58f24ce71fca95696dd2f9a083077a6ff365afb/frontend/src/console/models.ts#L41

You don't need to worry about merging things here, so just return the entire clean state.
```suggestion
      return initialConsoleState;
```

Reply:
Thanks for the suggestion, I have refactored the code :)